File: mojo/environment/test_support.cc
FASTVC-BENCH-CORPUS:chromium-main-v1-943b94ae-1c74-4335-94fa-ceb4d277cea8
// Copyright 2014 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

#include "mojo/public/environment/test_support.h"

#include "base/test/perf_log.h"

namespace mojo {
namespace test {

void LogPerfResult(const char* test_name, double value, const char* units) {
  return base::LogPerfResult(test_name, value, units);
}

}  // namespace test
}  // namespace mojo

File: chrome/browser/ash/system_extensions/api/test_support/test_support.js
FASTVC-BENCH-CORPUS:chromium-main-v1-943b94ae-1c74-4335-94fa-ceb4d277cea8
// Copyright 2022 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

// Script that imports all necessary scripts to run a test.
importScripts(
  'testharness.js',
  // Needed for using Mojo bindings.
  'mojo_bindings_lite.js',
  'system_extensions_test_runner.test-mojom-lite.js',
  // Registers callback that notify the browser of test harness events, e.g.
  // when tests finish running.
  'testharnessreport.js');
